core/transport/src/main/proto/grpc/file_system_master.proto

```diff
@@ -169,6 +169,18 @@ message FreePRequest {
   optional FreePOptions options = 2;
 }
 
+message FreeWorkerPResponse {}
+message FreeWorkerPOptions {
+  optional bool forced = 1;
+  optional int64 timeOut = 2;
+  optional FileSystemMasterCommonPOptions commonOption = 3;
```

Comment:
`FileSystemMasterCommonPOptions` is useless and I recommend moving this RPC to `block_master.proto`.